custom/aaa/models.py

```diff
@@ -691,14 +691,14 @@ def agg_from_growth_monitoring_forms_ucr(cls, domain, window_start, window_end):
                    array_agg(zscore_grading_wfa) AS zscore_grading_wfa_history
             FROM (
                 SELECT child_health_case_id,
-                       ARRAY[timeend::text, weight_child] AS weight_child,
-                       ARRAY[timeend::text, height_child] AS height_child,
+                       ARRAY[timeend::text, weight_child::text] AS weight_child,
```

Comment:
why are these text?

Reply:
also wondered that, would it not be better to store them as int?

Reply:
Arrays only support one type of value https://www.postgresql.org/docs/9.6/arrays.html

I've gone a bit back and forth on this and I think the best way to handle these history tables is to save it as text and perform the type cast in python. These are only being used for the history of the child's height, so they aren't being queried on.

Another solution would be having a generic table like:

| case_id | timeend | case property | value |
| ---------|-------------|------------------|-------- |
| child_health_1 |  2019-01-01 | weight_child | 5 |

But it felt very complex since these wouldn't be used for further aggregations (only getting one child at a time)